.gitattributes

```diff
@@ -1,10 +1,10 @@
 ## Platform files generated by Flutter during `flutter create`
-**/example/android/ linguist-generated=true
-**/example/ios/ linguist-generated=true
-**/example/linux/ linguist-generated=true
-**/example/macos/ linguist-generated=true
-**/example/web/ linguist-generated=true
-**/example/windows/ linguist-generated=true
+**/example/android/** linguist-generated=true
```

Comment:
I am guessing you pulled this commit in from next to get the unit tests passing. Can you also pull it into `feat/next/analytics`? I am not sure if there are other commits in this PR that are the same. You might want to rebase this branch and `feat/next/analytics`.